Dockerfile.ako

```diff
@@ -10,5 +10,6 @@ RUN GOARCH=amd64 CGO_ENABLED=0 GOOS=linux go build -o $GOPATH/bin/akc -mod=vendo
 FROM photon:latest
 RUN yum install -y tar.x86_64
 COPY --from=build /go/bin/akc .
+COPY ./scripts/pre_stop_hook.sh /var/pre_stop_hook.sh
```

Comment:
@sudswasavi : Can we avoid this hardcoding of path and mention relative path? as Dockerfile.ako will be running from AKO main folder so I didn't try to mention relative path

Reply:
Yes feel free to change in another PR. I will go ahead and merge this one.